demisto_sdk/commands/download/downloader.py

```diff
@@ -380,6 +375,20 @@ def build_custom_content_object(file_path: str) -> dict:
             'file_ending': file_ending,
         }
 
+    @staticmethod
+    def file_type_to_entity(file_data: dict, file_type: str) -> str:
+        """
+        Given the file type returns the file entity
+        :param file_data: The file data
+        :param file_type: The file type, for example: integration
+        :return: The file entity, for example: Integrations
+        """
+        if file_type and file_type == 'playbook':
+            name: str = get_entity_name_by_entity_type(file_data, PLAYBOOKS_DIR)
+            if name and 'test' in name.lower():
+                return TEST_PLAYBOOKS_DIR
```

Comment:
What about beta integrations?

Reply:
@Itay4 said a Pack doen't have the "Beta_Integrations" dir, so need to handle it in a special way like TestPlaybooks.